pkg/agent/openflow/pipeline.go

```diff
@@ -873,6 +889,47 @@ func (c *client) arpNormalFlow(category cookie.Category) binding.Flow {
 		Done()
 }
 
+func (c *client) conjunctionAllowMetricFlows(conjunctionID uint32, ingress bool) []binding.Flow {
+	metricTableID := IngressMetricTable
+	offset := 0
+	rng := binding.Range{0, 31}
+	if !ingress {
+		metricTableID = EgressMetricTable
+		offset = 32
+		rng = binding.Range{32, 63}
+	}
+	return []binding.Flow{
+		c.pipeline[metricTableID].BuildFlow(priorityNormal).MatchProtocol(binding.ProtocolIP).
```

Comment:
need to update ovs-pipeline.md and doc .. can be a TODO

Reply:
Added.